js/oai.ts

```diff
@@ -69,6 +72,7 @@ export function buildOpenAIClient(options: OpenAIAuth): OpenAI {
 }
 
 declare global {
+  // eslint-disable-next-line
```

Comment:
is there a more specific warning we can disable? or maybe a comment to explain what we are ignoring?

Reply:
Yep added more specific one